src/main/java/com/redhat/devtools/intellij/common/listener/TreeDoubleClickListener.java

```diff
@@ -31,10 +31,9 @@ public TreeDoubleClickListener(final JTree tree) {
         installOn(tree);
     }
 
-
     @Override
     protected boolean onDoubleClick(MouseEvent event) {
-        final TreePath clickPath = tree.getUI() instanceof WideSelectionTreeUI ? tree.getClosestPathForLocation(event.getX(), event.getY())
+        final TreePath clickPath = tree.getUI() instanceof DefaultTreeUI ? tree.getClosestPathForLocation(event.getX(), event.getY())
```

Comment:
@lstocchi dare to please test this change (you committed it)?

Reply:
Tried with tekton, works fine